.gitignore

```diff
@@ -90,3 +90,4 @@ NuGetExeBuild/
 NuGetRefBuild/
 Packages/
 NuGetTaskBuild/
+build/
```

Comment:
It's still a little bit secret if someone adds a new file, but I don't have a strong opinion about it. One way would be to have all artifacts in a separate folder .e.g. .artifacts/ but that's probably overkill in this case. Another option is just to exclude the new folder new folder.